packages/go/index.ts

```diff
@@ -94,332 +99,458 @@ export async function build({
   workPath,
   meta = {},
 }: BuildOptions) {
-  if (process.env.GIT_CREDENTIALS) {
-    debug('Initialize Git credentials...');
-    await initPrivateGit(process.env.GIT_CREDENTIALS);
-  }
+  const goPath = await getWriteableDirectory();
+  const srcPath = join(goPath, 'src', 'lambda');
+  const downloadPath = meta.skipDownload ? workPath : srcPath;
+  await download(files, downloadPath, meta);
```

Comment:
Download should mostly be a no-op because `skipDownload` is always true, according to @TooTallNate. I checked that this was the case locally and in CI.

Reply:
In a future PR, we can probably remove download() from all the builders